src/_standard.scss

```diff
@@ -4,19 +4,18 @@
 ////
 
 @mixin oQuoteStandardIcon {
-	position: absolute;
-	top: oTypographySpacingSize($units: 4);
-	left: oTypographySpacingSize($units: 2);
-	@include oIconsGetIcon('speech-left', oColorsGetPaletteColor('claret-80'), 50, $iconset-version: 1);
+	@warn "oQuoteStandardIcon is now deprecated and will be removed in next major release";
```

Comment:
Typo alert, missing "the": oQuoteStandardIcon is now deprecated and will be removed in the next major release.